NineChronicles.RPC.Shared/Services/IBlockChainService.cs

```diff
@@ -12,6 +12,8 @@ public interface IBlockChainService : IService<IBlockChainService>
         UnaryResult<byte[]> GetState(byte[] addressBytes);
 
         UnaryResult<byte[]> GetBalance(byte[] addressBytes, byte[] currencyBytes);
+        
+        UnaryResult<byte[]> GetTip(byte[] addressBytes);
```

Comment:
is this return `Block<T>`?

Reply:
I wanted `Unary Result(delayed Render?.Tip?.Header.Serialize());` as the result value.

Reply:
> is this return `Block<T>`?

I wanted Unary Result(delayed Render?.Tip?.Header.Serialize()); as the result value.